_posts/2016-02-04-ombulabs-products.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Announcing The Ombu Labs Shop!"
+date: 2016-02-04 18:35:00
+categories: ["Design"]
+author: "cecilia"
+---
+
+We recently worked on a series of Christmas presents for the team. We are happy to share part of the process and its final result!
+
+<!--more-->
+
+### Our process: it all started with an idea
+
+As it happens with every product, it starts with an idea. We wanted to make a t-shirt, a notebook and a tote bag that could reflect our passion for work and design.
+
+For the notebook, we chose a quote by Otl Aicher’s: “Good Art Inspires, Good Design Motivates”. Here you can see part of the design process, with some tweaks and final adjustments.
```

Comment:
@cecilia-ombulabs please correct this like so: "a quote by Otl Aicher" or "Otl Aicher's quote"